standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnDbUtil.java

```diff
@@ -180,6 +180,27 @@ public static void prepDb(Configuration conf) throws Exception {
         " WS_COMMIT_ID bigint NOT NULL," +
         " WS_OPERATION_TYPE char(1) NOT NULL)"
       );
+
+      stmt.execute("CREATE TABLE REPL_TXN_MAP (" +
+          " RTM_REPL_POLICY varchar(256) NOT NULL, " +
+          " RTM_SRC_TXN_ID bigint NOT NULL, " +
+          " RTM_TARGET_TXN_ID bigint NOT NULL, " +
+          " PRIMARY KEY (RTM_REPL_POLICY, RTM_SRC_TXN_ID))"
+      );
+
+      stmt.execute("CREATE TABLE \"APP\".\"SEQUENCE_TABLE\" (\"SEQUENCE_NAME\" VARCHAR(256) NOT " +
+              "NULL, \"NEXT_VAL\" BIGINT NOT NULL)"
+      );
+
+      stmt.execute("CREATE TABLE \"APP\".\"NOTIFICATION_SEQUENCE\" (\"NNI_ID\" BIGINT NOT NULL, " +
```

Comment:
If these CREATE TABLE command fails, then we rollback the txn and hence rest of the txn relevant tables won't be created as well. So, it is suggested to create it if not exists. If "IF NOT EXISTS" not supported, then catch the specific exception from these set of CREATE TABLE commands and handle differently. Also, move INSERT commands also within this try-catch block of these CREATE commands.